code/game/objects/items/devices/scanners.dm

```diff
@@ -264,58 +271,20 @@ GENE SCANNER
 							<td><font color='#00cc66'><b>[CEILING(tox_loss,1)]</b></font></td>\
 							<td><font color='#33ccff'><b>[CEILING(oxy_loss,1)]</b></font></td></tr>"
 
-			for(var/o in damaged)
-				var/obj/item/bodypart/org = o //head, left arm, right arm, etc.
-				dmgreport += "<tr><td><font color='#cc3333'>[capitalize(org.name)]:</font></td>\
-								<td><font color='#cc3333'>[(org.brute_dam > 0) ? "[CEILING(org.brute_dam,1)]" : "0"]</font></td>\
-								<td><font color='#ff9933'>[(org.burn_dam > 0) ? "[CEILING(org.burn_dam,1)]" : "0"]</font></td></tr>"
+			if(mode == SCANNER_VERBOSE)
+				for(var/o in damaged)
+					var/obj/item/bodypart/org = o //head, left arm, right arm, etc.
```

Comment:
**NOTE:** this suggestion changes the var name, you'll need to change other uses of `org` > `limb`. org is not a good var name!

```suggestion
				for(var/obj/item/bodypart/limb as anything in damaged)
```